packages/connect-node-test/src/helpers/testserver.ts

```diff
@@ -318,8 +317,23 @@ vKy9wyvtUtg=
         },
         useBinaryFormat: false,
       }),
-
-    // gRPC-web
+    "@bufbuild/connect-node (Connect, JSON, http) against connect-go (h1)": (
```

Comment:
Now that we have the first http1 client, can you also add transports that test against the Node.js http1 server? 


```ts
    "@bufbuild/connect-node (Connect, binary, http1) against @bufbuild/connect-node (h1)":
      (options?: Record<string, unknown>) =>
        createConnectHttp2Transport({
          ...options,
          baseUrl: servers["@bufbuild/connect-node (h1)"].getUrl(),
          useBinaryFormat: true,
        }),
    "@bufbuild/connect-node (Connect, JSON, http1) against @bufbuild/connect-node (h1)":
      (options?: Record<string, unknown>) =>
        createConnectHttp2Transport({
          ...options,
          baseUrl: servers["@bufbuild/connect-node (h1)"].getUrl(),
          useBinaryFormat: false,
        }),
```

If it doesn't work out of the box, let's sync. It's very possible that the http1 _server_ isn't working correctly - this will be the first time we test it.